<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogLevelDialog</class>
 <widget class="QWidget" name="LogLevelDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log level bits</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="allOn">
         <property name="text">
          <string>Mind be</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="allOff">
         <property name="text">
          <string>Mind ki</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="checkBoxException">
           <property name="text">
            <string>Üzenet kizáráskor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxError">
           <property name="text">
            <string>Hiba üzenetek</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxWarn">
           <property name="text">
            <string>Figyelmeztető üzenetek</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxInfo">
           <property name="text">
            <string>Informásiós üzenetek</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxVerb">
           <property name="text">
            <string>Részletes információk</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxVVerb">
           <property name="text">
            <string>Részletesebb információk</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxEnterLeave">
           <property name="text">
            <string>Függvényhívások követése</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxPArg">
           <property name="text">
            <string>Paraméterek kifejtése</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxSQL">
           <property name="text">
            <string>SQL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxObject">
           <property name="text">
            <string>Object new/delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxAddress">
           <property name="text">
            <string>Címek</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkBoxLv2">
           <property name="text">
            <string>library: lv2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxLv2d">
           <property name="text">
            <string>app: lv2d</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxLv2gui">
           <property name="text">
            <string>app: lv2gui</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxIcontsrv">
           <property name="text">
            <string>app:icontsrv</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxImport">
           <property name="text">
            <string>app:import</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxPortstat">
           <property name="text">
            <string>app:portstat</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="Cancel">
         <property name="text">
          <string>Elvet</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Ok">
         <property name="text">
          <string>O.K.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>allOn</tabstop>
  <tabstop>allOff</tabstop>
  <tabstop>checkBoxError</tabstop>
  <tabstop>checkBoxWarn</tabstop>
  <tabstop>checkBoxInfo</tabstop>
  <tabstop>checkBoxVerb</tabstop>
  <tabstop>checkBoxVVerb</tabstop>
  <tabstop>checkBoxEnterLeave</tabstop>
  <tabstop>checkBoxSQL</tabstop>
  <tabstop>Cancel</tabstop>
  <tabstop>Ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
